app/src/main/java/edu/uc/it3048/mypokedex/MainActivity.kt

```diff
@@ -4,20 +4,19 @@ import android.content.Intent
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import android.widget.ImageButton
-import edu.uc.it3048.mypokedex.ui.main.MainFragment
 
 class MainActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.main_activity)
 
-        // Calling the login method
+        /** Calling the login method */
         login()
     }
 
-    // Method to login to firebase, proceed to new screen
-    private fun login(){
+    /** Method to login to firebase, proceed to new screen */
```

Comment:
This is above a function, where KDoc belongs, but use the proper KDoc formatting and syntax: https://kotlinlang.org/docs/reference/kotlin-doc.html